src/includes/home/banner.njk

```diff
@@ -4,71 +4,39 @@
       <div class="p-2 rounded-lg bg-blue-700 shadow-lg sm:p-3">
         <div class="flex items-center justify-between flex-wrap">
           <div class="w-0 flex-1 flex items-center">
-            <a href="/blog/powering-community-led-growth/">
-            <span class="flex p-2 rounded-lg bg-blue-800">
-              <!-- Heroicon name: heart -->
-              <svg
-                class="h-6 w-6 text-white"
-                xmlns="http://www.w3.org/2000/svg"
-                fill="none"
-                viewBox="0 0 24 24"
-                stroke="currentColor"
-              >
-                <path
-                  stroke-linecap="round"
-                  stroke-linejoin="round"
-                  stroke-width="2"
-                  d="M4.318 6.318a4.5 4.5 0 000 6.364L12 20.364l7.682-7.682a4.5 4.5 0 00-6.364-6.364L12 7.636l-1.318-1.318a4.5 4.5 0 00-6.364 0z"
-                />
-              </svg>
-            </span>
+            <a href="/careers/">
+              <span class="flex p-2 rounded-lg bg-blue-800">
+                <!-- Heroicon name: heart -->
+                <svg class="h-6 w-6 text-white" xmlns="http://www.w3.org/2000/svg" fill="none" viewbox="0 0 24 24" stroke="currentColor">
+                  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4.318 6.318a4.5 4.5 0 000 6.364L12 20.364l7.682-7.682a4.5 4.5 0 00-6.364-6.364L12 7.636l-1.318-1.318a4.5 4.5 0 00-6.364 0z"/>
+                </svg>
+              </span>
             </a>
             <p class="ml-3 font-medium text-white truncate">
               <span class="md:hidden">
-                <a href="/blog/powering-community-led-growth/">
-                We’ve raised a $4M seed round
+                <a href="/careers/">
+                  Come grow with us! We're looking for awesome people to join the Orbit team!
```

Comment:
@alexlsalt I see the previous text was a shorter version of the expanded text after. I'm not sure if that's necessary but just thought I'd flag it.

Reply:
Totally right - updated in the latest commit. Thank you!